### PR TITLE
Sort out `pen`[`cil`]/`edit` icons/add `pencil-ruler` (stationery) icon/improve metadata

### PR DESCRIPTION
**Pull request by danielbayley**
Meant to represent tools/utilities/design/illustration/technical drawing/sketch etc…

Alternatives (including possible alterations to [`pencil`](https://lucide.dev/icon/pencil)…)

Untitled

Artboard 1@2x

### CODE REVIEW

**Comment by jguddas** [comment]
Maybe a bit big and maybe a bit hard to point out what the icon is supposed to convey.

**Comment by danielbayley** [comment]
> Maybe a bit big

It’s using exactly the proportions of `ruler` and `pen`[`cil`]…

> a bit hard to point out what the icon is supposed to convey.

> Meant to represent tools/utilities/design/illustration/technical drawing/sketch etc…

**Review by ericfennis — COMMENTED** [review]
Nice icon! 
Not sure about the name: `tools`, I think we can better name it as `ruler-pencil` or `pencil-ruler`.

**Comment by danielbayley** [comment]
> Not sure about the name: `tools`, I think we can better name it as `ruler-pencil` or `pencil-ruler`.

@ericfennis I think `pencil-ruler`, since the pencil is the front/overlapping element… A similar approach to the naming in #1156.

**Comment by jguddas** [comment]
What about naming it `stationery`?

**Comment by danielbayley** [comment]
> What about naming it `stationery`?

I think `pencil-ruler` is more in line with the general approach to naming. But that should definitely be added to `tags`…

**Comment by jguddas** [comment]
IMO `pencil-eraser` should become the new `pencil` and have an alias as `edit-2`.

`pencil-eraser` and `edit-2` could then be removed.

**Comment by jguddas** [comment]
`pencil-line` should also have the eraser IMO.

**Comment by danielbayley** [comment]
> I think we can have both `pencil` and `pencil-eraser`

Surely it should be `pen` and `pencil` then…?

> > pencil-line should also have the eraser IMO.

> Yes, I wholeheartedly agree, without the eraser, it looks a bit too much like an airplane.

I think @jguddas was referring to what is currently named `edit-3`…?

**Comment by danielbayley** [comment]
@karsa-mistmere @jguddas @ericfennis See latest changes…

**Review by ericfennis — APPROVED** [review]

File: icons/ruler.json [hunk 1]
```diff
@@ -8,9 +8,13 @@
...
   "tags": [
     "measure",
+    "centimetres",
     "meter",
     "foot",
-    "inch"
+    "feet",
+    "inches",
+    "units",
+    "stationery"
```
Comment (karsa-mistmere) [inline icons/ruler.json:17]: Maybe this (and also the pencil one?) could have some more tags, i.e. length, width, depth, breadth, dimensions, size, extent, measurements.

Comment (karsa-mistmere) [inline icons/ruler.json:17]:
```suggestion
    "stationery",
    "length",
    "width",
    "depth",
    "breadth",
    "dimensions",
    "size",
    "extent",
    "measurements"
```

File: icons/pencil-ruler.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "writing",
+    "stationery",
+    "artist",
+    "measure",
+    "centimetres",
+    "meter",
+    "foot",
+    "feet",
+    "inches",
+    "units"
```
Comment (karsa-mistmere) [inline icons/pencil-ruler.json:22]:
```suggestion
    "units",
    "length",
    "width",
    "depth",
    "breadth",
    "dimensions",
    "size",
    "extent",
    "measurements"
```

**Review by karsa-mistmere — APPROVED** [review]

**Comment by karsa-mistmere** [comment]
Nice work, @danielbayley!

**Pull request merged**